rust-utils/src/launcher_lair_client.rs

```diff
@@ -0,0 +1,331 @@
+        }
+
+        // Get or generate key for encryption of the seed
+        let encryption_key = match self
+            .lair_client
+            .get_entry("import-encryption-key".into())
+            .await
+        {
+            Ok(key) => match key {
+                LairEntryInfo::Seed { tag: _, seed_info } => seed_info,
+                _ => {
+                    return Err(napi::Error::from_reason(
+                        "The import encryption key in lair is of the wrong format.",
+                    ))
+                }
+            },
+            Err(_) => self
+                .lair_client
+                .new_seed("import-encryption-key".into(), None, false)
+                .await
+                .map_err(|e| {
+                    napi::Error::from_reason(format!(
+                        "Failed to create new import encryption key in lair: {}",
+                        e
+                    ))
+                })?,
+        };
+
+        // Get or generate key for decryption of the seed
+        let decryption_key = match self
+            .lair_client
+            .get_entry("import-decryption-key".into())
+            .await
+        {
+            Ok(key) => match key {
+                LairEntryInfo::Seed { tag: _, seed_info } => seed_info,
+                _ => {
+                    return Err(napi::Error::from_reason(
+                        "The import decryption key in lair is of the wrong format.",
+                    ))
+                }
+            },
+            Err(_) => self
+                .lair_client
+                .new_seed("import-decryption-key".into(), None, false)
+                .await
+                .map_err(|e| {
+                    napi::Error::from_reason(format!(
+                        "Failed to create new import decryption key in lair: {}",
+                        e
+                    ))
+                })?,
+        };
+
+        // encrypt seed that shall be imported
+        let (nonce, cipher) = self
+            .lair_client
+            .crypto_box_xsalsa_by_pub_key(
+                encryption_key.x25519_pub_key.clone(),
+                decryption_key.x25519_pub_key.clone(),
+                None,
+                derived_key_pair.secret.as_ref().into(),
+            )
+            .await
+            .map_err(|e| {
+                napi::Error::from_reason(format!("Failed to encrypt the device_bundle_seed: {}", e))
+            })?;
+
+        let src_tag = format!("imported#{initial_host_pub_key_b64_derived}");
+
+        // import the encrypted seed into lair
+        let imported_seed = self
+            .lair_client
+            .import_seed(
+                encryption_key.x25519_pub_key,
+                decryption_key.x25519_pub_key,
+                None,
+                nonce,
+                cipher,
+                src_tag.clone().into(),
+                false,
```

Comment:
Might there be a reason for having the seed be exportable? And are there any trade-offs to having it exportable (e.g. security-wise)?